tests/migrations/test_commands.py

```diff
@@ -528,16 +530,10 @@ def test_showmigrations_plan_app_label_error(self):
         no migrations are present in provided app labels.
         """
         # App with no migrations.
-        with self.assertRaisesMessage(CommandError, 'No migrations present for: unmigrated_app'):
-            call_command('showmigrations', 'unmigrated_app', format='plan')
-        # Nonexistent app (wrong app label).
-        with self.assertRaisesMessage(CommandError, 'No migrations present for: nonexistent_app'):
-            call_command('showmigrations', 'nonexistent_app', format='plan')
-        # Multiple nonexistent apps; input order shouldn't matter.
-        with self.assertRaisesMessage(CommandError, 'No migrations present for: nonexistent_app1, nonexistent_app2'):
-            call_command('showmigrations', 'nonexistent_app1', 'nonexistent_app2', format='plan')
-        with self.assertRaisesMessage(CommandError, 'No migrations present for: nonexistent_app1, nonexistent_app2'):
-            call_command('showmigrations', 'nonexistent_app2', 'nonexistent_app1', format='plan')
+        err = io.StringIO()
+        with self.assertRaises(SystemExit):
+            call_command('showmigrations', 'unmigrated_app', format='plan', stderr=err)
+        self.assertIn('No migrations present for: unmigrated_app', err.getvalue())
```

Comment:
This should be something like `self.assertEqual('migrations\n (no migrations)\n', out.getvalue().lower())` (perhaps adjusted a bit to match the output of showmigrations for other apps) (no SystemExit).